src/types/Contact.ts

```diff
@@ -24,7 +24,7 @@ export enum Periodicity {
     BI_MONTHLY = 'bi-monthly',
     QUARTERLY = 'quarterly',
     ANNUALLY = 'annually',
-    BI_ANUALLY = 'bi-anually',
+    BI_ANNUALLY = 'bi-annually',
```

Comment:
> `BI_ANNUALLY`

The constant name cannot be renamed under 7.x. As it's a BC break. Let's only change the value here.

Reply:
Oh, right, my bad

Reply:
https://github.com/prezly/javascript-sdk/pull/160/commits/039ce32f80a029ffd39b1dec0ec43eca57341c31